### PR TITLE
Add Slido URL option to a Presentation

### PR DESCRIPTION
**Pull request by pdamonkey**
Adds a new optional Slide URL entry for a Presentation and passes this back if it exists to the schedule API response.  
Also added `release: Run migrate -y` to the `Procfile` to allow migrations to happen when releasing.

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1 +1,2 @@
+release: Run migrate -y
```
Comment (SwiftyAlex) [inline Procfile:1]: Do we need this explicitly? we've been running auto migrations as far as I aware, so they should just run 🤔 

Me and @adamoxley both made changes that migrated I think

File: Resources/Views/Authentication/slot_form.leaf [hunk 2]
```diff
@@ -38,7 +38,7 @@
                 <select name="activityID" class="form-control activity">
                 <option disabled selected value> -- select an option -- </option>
                 #for(activity in activities):
-                  <option #if(activities.id == slot.activity.id): selected #endif id="activityID" name="activityID" value=#(activity.id)>
+                  <option #if(activity.id == slot.activity.id): selected #endif id="activityID" name="activityID" value=#(activity.id)>
```
Comment (SwiftyAlex) [inline Resources/Views/Authentication/slot_form.leaf:41]: good spot!

**Review by SwiftyAlex — APPROVED** [review]
Seems chill to go in, not too sure if we need the migrate manually

Are we able to create these ahead of time and then utilise slido to manage the scheduling? I.e set them all up the day before, upload the links to the website, but only actually have them be "askable" at the correct time? 

Just trying to make sure we can kinda let it do its thing on the day as that would be ideal!

**Comment by pdamonkey** [comment]
The migrations aren't running at the moment so adding that command to see if it resolves it. I've spoken about delaying Slido links, maybe only show them on the day

**Comment by adamoxley** [comment]
> The migrations aren't running at the moment so adding that command to see if it resolves it. I've spoken about delaying Slido links, maybe only show them on the day

Do we know why they weren't running? Was there any output within CI such as "no migrations to run" kind of thing? 

Automigrations worked last time I added new migrations. I'm not sure if there's a difference between automigrations and the command at the Vapor level.

**Comment by pdamonkey** [comment]
It didn't run on the last merge so hoping this does it. I couldn't see an automerge command in configure so not sure how it would do it normally.

**Comment by adamoxley** [comment]
> It didn't run on the last merge so hoping this does it. I couldn't see an automerge command in configure so not sure how it would do it normally. 

Ah I just checked the migrate file on main, looks like it might have been because the migration was added (line 76) after the auto migrate (line 70) so at that point, automigrate has already ran so it won't have ran the push notifications migration. 

It will run in order of what is prior to calling automigrate, but only migrate migrations that haven't been ran previously (fluent_migrations table in the DB keeps a record of what got ran and when).

I think this should migrate correctly if the migration is moved up above automigrate line. 

https://github.com/SwiftLeeds/swiftleeds-web/blob/main/Sources/App/Migrations.swift

**Pull request merged**